templates/3b_extract_images_from_bin.ipynb

```diff
@@ -10,7 +10,7 @@
     "# Extract Bin Files\n",
```

Comment:
<p>Remove from cell, documentation, and set as default for both this notebook and watcher extraction</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>